src/Reflection/ReflectionProperty.php

```diff
@@ -475,4 +564,23 @@ private function assertObject(mixed $object): object
 
         return $object;
     }
+
+    private function setModifiers(PropertyNode $node): void
```

Comment:
Probably best to name this `computeModifiers()`, and avoid the state mutation

Reply:
Fixed.